Updated screenshot and mention that expiration for Personal Access Token can be chosen when creting a new token (#523)
okteto/docs: src/content/cloud/personal-access-tokens.mdx

### Token Expiration

> When you create a token, you can select the expiration, by default is 180 days. Once a token has expired, it can no longer be used to authenticate and for API requests. It is not possible to restore an expired token, you will need to create a new token upon expiration.

Reviewer: ```suggestion
When you create a token, you can select the expiration, the default expiration is 180 days. Once a token has expired, it can no longer be used to authenticate or for API requests. It is not possible to restore an expired token, you will need to create a new token upon expiration.
```
